source/material_model/interface.cc

```diff
@@ -852,6 +863,19 @@ namespace aspect
 
 
 
+    template<int dim>
+    std::vector<double>
+    NamedAdditionalMaterialOutputs<dim>::get_nth_output(const unsigned int idx) const
+    {
+      // In this function we extract the values for the nth output
+      // The number of outputs is the outer vector
+      Assert (output_values[idx].size() > 0,
```

Comment:
Yes, I think you need another Assert for `output_values.size() > idx` just to be sure.